files/image_config/updategraph/updategraph

```diff
@@ -26,21 +49,18 @@ fi
 
 . /etc/sonic/updategraph.conf
 
+check_system_warm_boot
+
 if [ -f /tmp/pending_config_migration ]; then
-    if [ "$enabled" = "true" ]; then
+    copy_config_files minigraph.xml snmp.yml acl.json config_db.json
```

Comment:
Any reason we don't copy the whole old_config directory to the new /etc/sonic?  
Not necessarily related to this PR, but in case of FRR split configuration, the frr configuration should be copied over too.

Reply:
I can see that we don't want to copy sonic_version.yml for sure.

I don't know the concern of not copying the rest. @taoyl-ms might know better.

Reply:
OK, let's address that in later PRs for other files need to be copied. and let this PR to address the warm-reboot related issues.

Reply:
Thanks!